farge

```diff
@@ -133,18 +135,20 @@ image_preview(){
 
 main() {
     if [ "$WAYLAND_DISPLAY" ]; then
-        HEX_COLOR=$(grim -g "$(slurp -p)" -t ppm - | convert - -format '%[pixel:p{0,0}]' txt:- | tail -1 | grep '#' | awk '{print $3}')
+        HEX_COLOR=$(grim -g "$(slurp -p)" -t ppm - |
+            convert - -format '%[pixel:p{0,0}]' txt:- | tail -n1 |
+            grep '#' | awk '{print $3}')
     else
         HEX_COLOR=$(colorpicker -doq)
     fi
-    
+
     # If no color is chosen, then exit
-    [[ ${#HEX_COLOR} -eq 0 ]] && echo "No color was selected" && exit 1
-    
+    [[ ! $HEX_COLOR ]] && echo "No color was selected" && exit 1
+
```

Comment:
You don't need to do `[ ${#var} -eq 0 ]`, you can just do `[ ! $var ]`, which checks if it exists, then reverses the exit signal.